src/main/java/alibek/User.java

```diff
@@ -0,0 +1,33 @@
+package alibek;
+
+import java.util.*;
+
+
+public class User {
+    String name;
+    User(String name) throws Exception {
+        validateName(name);
+        this.setName(name);
+    }
+    User() {
+
+    }
+    public String getName() {
+        return this.name;
+    }
+    public void setName(String name) {
```

Comment:
&#x1F34E; **Issue**
You probably don't need this setter, if you initialize name through constructor and never change it.
Setters are usually considered to be a bad practice and should be avoided (as they allow modifications of the object's state at any place in your project, better not allow you object to change as long as possible).